internal/evaluator/conftest_evaluator.go

```diff
@@ -163,23 +163,42 @@ type conftestRunner struct {
 }
 
 func (r conftestRunner) Run(ctx context.Context, fileList []string) (result []Outcome, data Data, err error) {
+	if log.IsLevelEnabled(log.TraceLevel) {
+		r.Trace = true
+	}
+
 	var conftestResult []output.CheckResult
 	conftestResult, err = r.TestRunner.Run(ctx, fileList)
 	if err != nil {
 		return
 	}
 
-	for _, r := range conftestResult {
+	for _, res := range conftestResult {
+		if log.IsLevelEnabled(log.TraceLevel) {
+			for _, q := range res.Queries {
+				for _, t := range q.Traces {
+					log.Tracef("[%s] %s", q.Query, t)
+				}
+			}
+		}
+		if log.IsLevelEnabled(log.DebugLevel) {
+			for _, q := range res.Queries {
+				for _, o := range q.Outputs {
+					log.Debugf("[%s] %s", q.Query, o)
```

Comment:
I don't see these showing up.

Reply:
Make sure to point (both of) your policy sources locally, e.g.:
```
  - policy:
      - file::.../ec-policies/policy/lib
      - file::.../ec-policies/policy/release
```
Though I could have added an acceptance test TBH...